mkdocs.yml

```
@@ -1,58 +1,58 @@
-            members_order: source
-            show_root_heading: true
-            heading_level: 3
-nav:
-  - Home: index.md
-  - API Reference:
-      - Overview: api.md
-      - Data Models (generated): generated/models.md
-      - JSON Schemas: schemas.md
-  - CLI Guide: cli.md
-  - MCP Server: mcp.md
+site_name: ExStruct
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)